coretex/cli/config.py

```diff
@@ -1,17 +1,26 @@
-from typing import Optional, Tuple
 from pathlib import Path
 
 from tabulate import tabulate
 
 import click
 
-from .utils import arrowPrompt
+from .utils import arrowPrompt, isGPUAvailable, validate
 from ..networking import networkManager
 from ..statistics import getAvailableRamMemory
 from ..configuration import loadConfig, saveConfig, isUserConfigured, isNodeConfigured
 
 
-def authenticate(retryCount: int = 0) -> Tuple[str, str, str, str]:
+class LoginInfo:
+    def __init__(self, username: str, password: str, token: str, tokenExpirationDate: str, refreshToken: str, refreshTokenExpirationDate: str):
```

Comment:
Convert `LoginInfo` into a `dataclass`. `dataclass` generates overhead like this `__init__` automatically.